caramel/models.py

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env python
 # vim: expandtab shiftwidth=4 softtabstop=4 tabstop=17 filetype=python :
+# pylint: disable=no-member
```

Comment:
Why is this one here?  file-wide disables should usually have a reasoning, and the commit message conflicts two different changes at once, making blame view less helpful.  

ie. the change is valid, but the documentation of it is less so, as having it in one file may cause others to just copy-paste it as a "valid fix" when there isn't a clear reason for why it's disabled.

Reply:
I'll move this to setup.cfg and restrict it to the effected areas.